python/src/main/resources/grpc/python/zeppelin_python.py

```diff
@@ -113,6 +113,29 @@ def show_dataframe(self, df, show_index=False, **kwargs):
     #)
     body_buf.close(); header_buf.close()
 
+  def registerHook(self, event, cmd, replName=None):
```

Comment:
hmm, I think we have doc on the python z context API, let's make sure we have them

Reply:
also new api is better to be in a separate PR